pyproject.toml

```diff
@@ -47,6 +47,8 @@ scikit-learn = "*"
 imblearn = "*"
 kaplanmeier = "*"
 pyinstaller = "*"
+lifelines = "0.29.0"

```

Comment:
lifelines is a dependency of kaplanmeier. the latest version of lifelines requires sciPy 2.x (which then triggers a lot of API failures).  Pinning at 0.29.0 moves it to the next to last version which is still compatible with everything else.  I don't recall if it required pandas 2.x, but that was probably the driver for updating the version of pandas (which then triggered the need to change the API call in Anonymise).

Reply:
agreed